examples/demo_bgp.pp

```diff
@@ -70,6 +70,7 @@
     bestpath_med_confed                    => true,
     bestpath_med_missing_as_worst          => true,
     bestpath_med_non_deterministic         => $bestpath_med_non_deterministic,
+    nsr                                    => false,
```

Comment:
As discussed in scrum this morning - this needs to be a conditional property since it only applies to XR.